test/e2e/network/dns.go

```diff
@@ -376,7 +384,13 @@ var _ = SIGDescribe("DNS", func() {
 		validateTargetedProbeOutput(f, pod3, []string{wheezyFileName, jessieFileName}, svc.Spec.ClusterIP)
 	})
 
-	ginkgo.It("should support configurable pod DNS nameservers", func() {
+	/*
+		Release: v1.16
+		Testname: DNS, custom dnsConfig
+		Description: Create a Pod with DNSPolicy as None and custom DNS configuration, specifying nameservers and serach path entries.
+		Pod creation MUST be successful and provided DNS configuration MUST be configured to /etc/resolve.conf file.
```

Comment:
Remove the comment about /etc/resolv.conf, it should read:

...MUST be configured in the Pod.

Reply:
Done.